snap/README.md

```diff
@@ -0,0 +1,43 @@
+# Kata Containers Snap
+
+This directory contains the resources needed to build the Kata Containers
+[snap][1] image.
+
+## Initial setup
+
+*Ubuntu Bionic*
```

Comment:
Could you either replace with the version number of add it (since the versions numbers are more meaningful than the names imho ;)